packages/suite/src/actions/suite/storageActions.ts

```diff
@@ -503,7 +495,7 @@ export const loadStorage = () => async (dispatch: Dispatch, getState: GetState)
 
 export const init = () => async (dispatch: Dispatch) => {
     // should be called only once
-    if (await isDBAccessible()) {
+    if (await db.isSupported()) {
```

Comment:
note to reviewers: previous code probably never worked since `isDBAccessible` was checking if db is blocked.
if it was blocked then `db.onBlocked` was never assigned and never triggered